### PR TITLE
Prefilled smart action

### PR DESCRIPTION
**Pull request by Tolsa**
- adding content/code/screenshot to all stacks

### CODE REVIEW

File: _includes/express-mongoose/actions.md [hunk 1]
```diff
@@ -235,6 +235,13 @@ function generateInvoice(request, response) {
   </div>
 </div>
 
+## Prefill a form with default values
+
+To save you a bit of time, Forest Admin allows you to prefill your form with values coming from elsewhere. In this example, we will prefill the form with informations coming from the record itself **(1)**{:.green}, with just a few extra lanes of code.
```
Comment (SeyZ) [inline _includes/express-mongoose/actions.md]: - You use 2 times the word "prefill". What about "Forest Admin allows you to set default values to your form". Something like that?
- "Extra lanes of code" typo ? --> "Extra lines of code"

File: _includes/express-mongoose/actions.md [hunk 2]
```diff
@@ -235,6 +235,13 @@ function generateInvoice(request, response) {
   </div>
 </div>
 
+## Prefill a form with default values
+
+Forest Admin allows you to set default values to your form. In this example, we will prefill the form with informations coming from the record itself **(1)**{:.green}, with just a few extra lines of code.
```
Comment (SeyZ) [inline _includes/express-mongoose/actions.md]: with data ?
or
with information ?

Comment (Tolsa) [reply _includes/express-mongoose/actions.md]: I will go with 'data' ! thx

**Comment by Tolsa** [comment]
> 'data' replaces 'informations' everywhere

**Pull request merged**